docs/user-guide/plugins.md

```diff
@@ -104,7 +104,11 @@ plugin "foo" {
 }
 ```
 
-When the plugin is enabled, TFLint invokes the `tflint-ruleset-[name]` (`tflint-ruleset-[name].exe` on Windows) binary in the plugin directory (For instance, `~/.tflint.d/plugins/tflint-ruleset-[name]`). So you should move the binary into the directory in advance.
+When enabled, TFLint looks for the plugin binary following the pattern tflint-ruleset-\[name] (or tflint-ruleset-\[name].exe on Windows) directly in your plugins folder (e.g., ~/.tflint.d/plugins/ on Unix-like systems or C:\Users\\[username]\\.tflint.d\plugins\ on Windows).
```

Comment:
By the way, to be honest, there is an issue with the location of files in Windows...
It should probably be placed in `%APPDATA%/tflint.d/plugins` like Terraform.
https://developer.hashicorp.com/terraform/cli/config/config-file#implied-local-mirror-directories